src/main/java/org/ngafid/accounts/EmailType.java

```diff
@@ -167,13 +188,14 @@ public static void insertEmailTypesIntoDatabase(int userIDTarget) {
             removeOldEmailTypesFromDatabase(currentEmailTypes);
         }
 
-
         StringBuilder query = new StringBuilder();
         query.append("INSERT INTO email_preferences (user_id, email_type) ");
 
         //Generate "individual" email type queries
         for (EmailType emailType : EmailType.values()) {
-            query.append("SELECT ").append(userIDTarget).append(", '").append(emailType.getType()).append("' UNION ALL ");
+            String emailTypeKey = emailType.getType();
+            emailTypeKeysRecent.add(emailTypeKey);
+            query.append("SELECT ").append(userIDTarget).append(", '").append(emailTypeKey).append("' UNION ALL ");
         }
 
         //Remove the last "UNION ALL" from the query
```

Comment:
Remove prints. If you have logging information you want to keep in production for our logs, use the logger.